src/WebJobs.Script.WebHost/Diagnostics/LinuxAppServiceEventGenerator.cs

```diff
@@ -8,13 +8,20 @@ namespace Microsoft.Azure.WebJobs.Script.WebHost.Diagnostics
 {
     internal class LinuxAppServiceEventGenerator : LinuxEventGenerator
     {
+        private readonly IEnvironment _environment;
+        private readonly Action<string> _writeEvent;
         private readonly LinuxAppServiceFileLoggerFactory _loggerFactory;
         private readonly HostNameProvider _hostNameProvider;
+        private string _tenantId;
+        private string _containerName;
 
-        public LinuxAppServiceEventGenerator(LinuxAppServiceFileLoggerFactory loggerFactory, HostNameProvider hostNameProvider)
+        public LinuxAppServiceEventGenerator(IEnvironment environment, LinuxAppServiceFileLoggerFactory loggerFactory, HostNameProvider hostNameProvider, Action<string> writeEvent = null)
         {
+            _writeEvent = writeEvent ?? WriteEvent;
+            _environment = environment;
             _loggerFactory = loggerFactory;
             _hostNameProvider = hostNameProvider ?? throw new ArgumentNullException(nameof(hostNameProvider));
+            _containerName = _environment.GetEnvironmentVariable(EnvironmentSettingNames.ContainerName)?.ToUpperInvariant();
```

Comment:
Looks like this was lifted from the LinuxContainerEventGenerator. We don't set these environment variables for Linux App Service.

Reply:
I wasn't sure if these fields were applicable to logs from apps running on Linux dedicated and/or premium. I'll remove them, as well as the corresponding placeholders from the log statement.